MobiFlight/MobiFlightCache.cs

```diff
@@ -303,10 +243,7 @@ public async Task<bool> connectAsync(bool force=false)
             {
                 if (!devInfo.HasMfFirmware()) continue;
 
-                MobiFlightModule m = new MobiFlightModule(new MobiFlightModuleConfig() {
-                    ComPort = devInfo.Port,
-                    Type = devInfo.Type
-                });
+                MobiFlightModule m = new MobiFlightModule(devInfo.Port, devInfo.Board);

```

Comment:
And this is the magic :) Assigning the board to the module here means that it's available on the module throughout the rest of the MobiFlight codebase.